doc/attr.xml

```diff
@@ -1774,3 +1774,85 @@ gap> ChromaticNumber(D);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphCartesianProductProjections">
+<ManSection>
+  <Attr Name="DigraphCartesianProductProjections" Arg="digraph"/>
+  <Returns>A list of transformations.</Returns>
+  <Description>
+    If <A>digraph</A> is a Cartesian product digraph, 
+    <C>digraph = DigraphCartesianProduct(gr_1, gr_2, ... )</C>, then
+    <C>DigraphCartesianProductProjections</C> returns a list <C>proj</C>
+    such that <C>proj[i]</C> is the projection onto the <C>i</C>-th
+    coordinate of the product.
+
+    A projection is an idempotent endomorphism of <A>digraph</A>. If
+    <C>gr1, gr2, ...</C> are all loopless digraphs, then the restriction
+    of <A>digraph</A> onto the image of <C>proj[i]</C> is isomorphic to
```

Comment:
Sorry, I can't work out immediately exactly what
```
the restriction of <A>digraph</A> onto the image of <C>proj[i]</C> is isomorphic to <C>gr_i</C>.
```
means. Could you try and explain, and then maybe we can work out a clearer way of phrasing it.

Reply:
Ah @reiniscirpons, do you mean `the induced subdigraph of <A>digraph</A>` on the image of <C>proj[i]</C>`? If so, then please update it to use `induced subdigraph`.